fixchain/hash.go

```diff
@@ -60,7 +60,7 @@ type bag struct {
 	certs []*x509.Certificate
 }
 
-// sort.Sort(data Interface) for bag - uses data.Len, data.Less & data.Swap
+// Len implements sort.Sort(data Interface) for bag - uses data.Len, data.Less & data.Swap
```

Comment:
This reads more like documentation for `bag` - move it up to that instead?

Reply:
Done.